Source/HTTPHeaders.swift

```diff
@@ -401,7 +401,7 @@ extension HTTPHeader {
 }
 
 extension Collection where Element == String {
-    var qualityEncoded: String {
+    func qualityEncoded() -> String {
```

Comment:
Swift guidelines recommend only O(1) computations for properties. It's not really an issue, but since this is more of a transform than a property of a `Collection` of `String`s, it seems more appropriate as a function.

Reply:
Thanks for the clarification. I figured there was some reasoning behind it.